src/main/java/uk/gov/hmcts/reform/judicialapi/service/impl/JudicialUserServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.judicialapi.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.judicialapi.controller.response.OrmResponse;
+import uk.gov.hmcts.reform.judicialapi.domain.UserProfile;
+import uk.gov.hmcts.reform.judicialapi.repository.UserProfileRepository;
+import uk.gov.hmcts.reform.judicialapi.service.JudicialUserService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
+@Service
+public class JudicialUserServiceImpl implements JudicialUserService {
+
+    @Autowired
+    private UserProfileRepository userProfileRepository;
+
+    private int defaultPageSize;
+
+
+    @Override
+    public ResponseEntity<Object> fetchJudicialUsers(Integer size, Integer page) {
+
+        Pageable pageable = createPageableObject(page, size);
+        Page<UserProfile> pagedUserProfiles = userProfileRepository.findAll(pageable);
+
+        List<UserProfile> userProfiles = pagedUserProfiles.getContent();
+
+        List<OrmResponse> ormResponses = userProfiles.stream()
+                .map(OrmResponse::new)
+                .collect(Collectors.toList());
+
+        return ResponseEntity
+                .status(200)
+                .body(ormResponses);
+    }
+
+    public Pageable createPageableObject(Integer page, Integer size) {
```

Comment:
you can move this to common util class to create generic pageable object and validate as well - you can refer https://github.com/hmcts/rd-caseworker-ref-api/blob/04b4cc3629642fb318c6c98f3f656dc23fc36acb/src/main/java/uk/gov/hmcts/reform/cwrdapi/util/RequestUtils.java#L45 if you want :)